data/remnant/remnant 2 side missions.txt

```diff
@@ -144,3 +144,244 @@ mission "Remnant: Visit to Paradise 1"
+		system Lucina
+		fleet "Korath Ember Waste Raid" 1
+	on complete
+		payment 250000
+		clear "Remnant: Shattered Light 3b: failed"
+		set "Remnant: Shattered Light 3b: done"
+		conversation
+			`	The Shattered Light is so massive that traffic control directs you to a large valley next to the spaceport, where you discover that a section of the chasm has been covered to create a semi-enclosed area. Judging from the equipment and stacks of freight, this area looks like it is predominantly used by bulk freight shipments and little else. Now, however, as the Shattered Light slowly moves in, a flock of Pelicans scatter in front of it looking like nothing more than a flock of plovers in front of an approaching camel.`
+			`	Down on the ground, you see Ruach and her team exit an airlock on the side of the Shattered Light and jump down to the ground. They gather around, giving hugs. "<first>!" exclaims Ruach. "We have recovered a lost piece of Remnant history. Thank you for helping. I do not know what this will mean for us in the long term, but it will be a lot of careful examination for the next few weeks."`
+			choice
+				`	"I have things to do, so just let me know if I can help."`
+					goto shatteredhelp
+				`	"So, what happened?"`
+			`	The team of Remnant are almost too excited for you to understand their gestures, but one calms down enough that their gestures are comprehensible. "To be honest, we were expecting this to be the first of several trips doing detailed scans and trying out different sorts of synchronization fields. But when we saw that it was heading to the planet..." She trails off, and another one continues. "We already had the sensors running as we approached, so we jumped and triggered the fields as soon as we got a reasonable match."`
+			`	He frowns a bit "It worked, although not perfectly. There are chunks missing, some of which seriously impair its functionality. But it is back in the primary weft, so now we have time to work on it and figure out what happened to it.`
+			choice
+				`	"I think I get the picture. Let me know if I can help."`
+					goto shatteredhelp
+				`	"What is this 'Primary Weft' you have mentioned?"`
+			`	Another Remnant steps forward. "The Weft is what we term the mesh of dimensions, energy, and time that form a subset of the universe. A layer, of sorts." He looks thoughtful. "What you probably know as 'Hyperspace' is what we technically call the secondary weft. There are a few others we are fairly confident exist, and several that we have hypothesized might exist but we have no means of confirming. Practically, each of these overlaps with the primary weft to some extent." His gestures take on an amused tone. "Of course, what we call the Primary Weft is only primary to those of us who live here. For all we know, there may be people living elsewhere that consider their own weft to be the primary one. Nothing says ours is special or the best or anything of the sort."`
```

Comment:
Alright, I don't have time to get into this in detail right now, but this is a lot of heavy cosmological lore to drop into a casual side mission. It is also a departure from the lore document despite pulling in a piece of terminology for it, used incorrectly.

This is not necessarily a problem, the lore document was only intended as MZ's notes and we're not _obliged_ to conform to it. It does; however, have some significant implications for the discussion around what hyperlanes are and how they work - with some departures from common assumptions.

On all accounts it at least warrants close scrutiny.

Reply:
Definitely. It is meant to provoke thought, after all. That being said, it is all presented as a mix of what they think it's true and what they hypothesize is true, so I don't think it should be a problem either way. :) They might be the most knowledgeable humans on the subject, but they are nowhere near the understanding of the Quarg or Pug our Korath.

Reply:
I've tweaked this reference a bit to add in a mention of the "warp" and introduce a bit of discussion into the matter. So now it isn't just the Remnant stating "this is the name," but rather them saying "This is what we've called it, but we're thinking this other name might fit better." In other words, it is a growing field.